tests/datasets/test_hooks.py

```diff
@@ -7,6 +7,7 @@
 from nussl.datasets import transforms
 import tempfile
 import shutil
+import librosa
```

Comment:
I needed it in the test I wrote to make sure the samples were salient, but I'm realized that test doesn't quite work. I expect I'll need it when I revise the test. For now it can be removed.